src/vmm/src/lib.rs

```diff
@@ -647,7 +693,10 @@ impl Vmm {
         Ok(())
     }
 }
-
+fn get_mmio_from_range(range: RangeInclusive) -> MmioRange {
+    //Address allocator will make sure we have the address available.
+    MmioRange::new(MmioAddress(range.start()), range.len()).unwrap()
```

Comment:
Are we sure this cannot fail?

Reply:
since we are passing the range, which we get from the address allocator, address allocator takes care of making sure there is always address available.